src/app/views/sidebar/resource-explorer/ResourceExplorer.tsx

```diff
@@ -183,7 +184,7 @@ const UnstyledResourceExplorer = (props: any) => {
 
   return (
     <section style={{ marginTop: '8px' }}>
-      {!isolated && <>
+      {!isolated && items[0].links.length > 0 && <>
```

Comment:
Is items always populated?

Reply:
![image](https://user-images.githubusercontent.com/34104277/216349737-302d5a63-a0bb-49b8-9b65-5eef06801abc.png)
This is what the condition is trying to preempt from rendering: the search box and the switch to beta toggle button.